### PR TITLE
Add Intune Section to Integration Doc

### LINKED ISSUE #436: List Intune on Integrations Page
## What New Or Updated Would You Like To See?
Wanna add Intune to our [integrations features page](https://docs.chocolatey.org/en-us/features/integrations). Since it isn't listed anywhere at https://chocolatey.org/compare currently. 

## Why Is It Needed?
Make note of our product integration outside of the setup docs themselves.

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
Add Intune section to integrations doc.

## Motivation and Context
No prior reference to intune integration besides the setup and use docs. Also brought page up to current linting standards.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [X] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated
* [X] No navigation changes needed

## Related Issue

Fixes #436

### CODE REVIEW

File: input/en-us/features/integrations.md [hunk 1]
```diff
@@ -216,3 +216,8 @@ Chocolatey integrates with SCCM by handling the software management, and pointin
 
 To direct Chocolatey package installs, you can still write GPOs to ensure this.
 
+## Microsoft Intune
+
+Chocolatey's Intune Integration is only available to our Chocolatey for Business customers. It allows you to take any Chocolatey package and `choco convert` them into an intunewin app. It then allows you to then `choco push` your converted Chocolatey packages to your intune tenant for hosting and distribution.
```
Comment (steviecoaster) [inline input/en-us/features/integrations.md:221]:
```suggestion
Chocolatey's Intune Integration is only available to our Chocolatey for Business customers. It allows you to take any Chocolatey package and `choco convert` it into an intunewin application. It then allows you to `choco push` your converted Chocolatey packages to your Intune tenant for hosting and distribution.

**Review by steviecoaster — CHANGES_REQUESTED** [review]

File: input/en-us/features/integrations.md (hunk 1 above)
Comment (pauby) [inline input/en-us/features/integrations.md:221]:
```suggestion
Chocolatey's Intune Integration is only available to our Chocolatey for Business customers. It allows you to take any Chocolatey package and convert them into an `.intunewin` format for pushing to your Intune tenant for hosting and distribution.
```

Comment (pauby) [reply input/en-us/features/integrations.md:221]: intunewin is not an actual thing, it's a format.

Comment (ryanrichter94) [reply input/en-us/features/integrations.md]: Fixed, thanks for the clarification.

Comment (ryanrichter94) [reply input/en-us/features/integrations.md]: Went with @pauby's recommendations as it seemed more concise. Thanks for the suggestions though.

**Review by pauby — APPROVED** [review]
LGTM. Just need to squash the commits, and we're good to merge from my perspective.

**Comment by gep13** [comment]
@ryanrichter94 am I right in saying that you used the "big green" button here, and squashed the commits through the UI?

**Comment by ryanrichter94** [comment]
Yes

**Pull request merged**